orderbook/src/database/events.rs

```diff
@@ -153,6 +154,34 @@ async fn insert_trade(
                 .bind(u256_to_big_decimal(&event.fee_amount)),
         )
         .await?;
+    // TODO: Temporarily insert_settlement here with the limited information we have
+    insert_settlement(transaction, index).await?;
```

Comment:
The insert settlement method could be slightly more general here if we spoof the index being passed in. We could also introduce the `SettlementEvent` Struct here and pass a spoofed/dummy value so the `insert_settlement` could already be written in its most general form.